content/en/synthetics/mobile_app_testing/mobile_app_tests/steps.md

```diff
@@ -166,6 +179,16 @@ For more information about advanced options for subtests, see [Advanced Options
 
 If it does not make sense for you to run your subtest independently, you can pause it. The test continues to be called as part of your parent test, and is not executed individually. For more information, see [Reusing Browser Test Journeys Across Your Test Suite][6].
 
+### Variables
+If your subtest contains variables, they are inherited by the test you import it into. 
```

Comment:
I'm not sure what "it" represents in this sentence, but if the meaning would be obvious to your audience, no changes needed. If your intention was to refer to the variables, I recommend using "them" instead of "it".

Reply:
ah yes great catch, I will change it to "them" :)